File: PRAK404-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def main():
    while True:
        print("Pilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")

        try:
            choice = int(input("Masukkan Pilihan: "))

            if 1 <= choice <= 4:
                num1 = float(input("Masukkan nilai pertama: "))
                num2 = float(input("Masukkan nilai kedua: "))

                if choice == 1:
                    result = num1 + num2
                    print(f"Hasil penjumlahan antara {num1:.2f} dengan {num2:.2f} adalah {result:.2f}")
                elif choice == 2:
                    result = num1 - num2
                    print(f"Hasil pengurangan antara {num1:.2f} dengan {num2:.2f} adalah {result:.2f}")
                elif choice == 3:
                    result = num1 * num2
                    print(f"Hasil perkalian antara {num1:.2f} dengan {num2:.2f} adalah {result:.2f}")
                elif choice == 4:
                    if num2 != 0:
                        result = num1 / num2
                        print(f"Hasil pembagian antara {num1:.2f} dengan {num2:.2f} adalah {result:.2f}")
                    else:
                        print("Error: Pembagian dengan nol tidak diizinkan")
            elif choice == 5:
                print("Terimakasih, telah menggunakan kalkulator NAMAANDA")
                break
            else:
                print("Input anda salah, silahkan coba lagi")
        except ValueError:
            print("Input harus berupa angka, silahkan coba lagi")

if __name__ == "__main__":
    main()

File: MODUL 4/PRAK404-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

int main() {
int choice;
float num1, num2, result;

do {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &choice);

if (choice >= 1 && choice <= 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &num1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &num2);

            switch (choice) {
                case 1:
                    result = num1 + num2;
                    printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", num1, num2, result);
                    break;
                case 2:
                    result = num1 - num2;
                    printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", num1, num2, result);
                    break;
                case 3:
                    result = num1 * num2;
                    printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", num1, num2, result);
                    break;
                case 4:
                    if (num2 != 0) {
                        result = num1 / num2;
                        printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", num1, num2, result);
                    } else {
                        printf("Error: Pembagian dengan nol tidak diizinkan\n");
                    }
                    break;
            }
        } else if (choice == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Rakhmad Fauzhan Ramadhan\n");
            break;
        } else {
            printf("Input anda salah, silahkan coba lagi\n");
        }
    } while (1);

    return 0;
}
